### ISSUE TITLE
botocove sometimes crashes when describing accounts

### ISSUE THREAD

**Issue by iainelder** [connelldave/botocove#12]
Sometimes I get an error like this when I run botocove:

```
TooManyRequestsException: An error occurred (TooManyRequestsException) when calling the DescribeAccount operation (reached max retries: 4): AWS Organizations can't complete your request because another request is already in progress. Try again later.
```

I'm running it in an organization with 915 accounts.

Can we do something to make it more robust here?

Do you need a complete example of what I'm doing? I haven't included example code here because I'm under the impression that this could happen with any query at the right scale.

**Comment by connelldave** [comment]
I've opened a PR with a quick fix for this, if you want to have a look :) This API call is pretty wasteful and in hindsight not very valuable, but I'd rather keep it and handle errors than potentially add a minor functionality change for now by removing it entirely.

**Comment by iainelder** [comment]
Makes sense! The changes look good to me. I more feedback on the PR itself.

I don't have a reliable way to reproduce the error.

When I get to use the new version, I'll look out for "Failed to call describe_account" messages and let you know how it goes.

**Comment by connelldave** [comment]
Released in 1.3.1 https://pypi.org/project/botocove/ - let me know if/when this causes something downstream to blow up instead :) I think this is a problem better addressed by better architecture going forward, which will take a bit of time and thought.
